### DISCUSSION TITLE
Load tcss from string instead of file

### DISCUSSION THREAD

**Discussion by PlasmaHH** [Textualize/textual#5283] [Q&A] [answered]
Hi,

I have been tinkering with textual a bit and been wondering if it would be possible (or maybe I overlooked it) to (re)load the tcss data from a string instead of a file?

**Comment by willmcgugan** [comment, marked as answer]
You can use the `CSS` classvar on App / Screen.

**Reply by PlasmaHH** [reply]
Ah thanks, that sems to work for the first start, but it won't be updated from that string when writing a new value?

**Reply by willmcgugan** [reply]
You would have to restart the app to update the CSS.

**Reply by PlasmaHH** [reply]
Hm, thanks, not ideal but doable...
